templates/mmm/degree_days.html

```diff
@@ -0,0 +1,159 @@
+{% extends 'base.html' %} {% block content %}
+<h3>Degree Days Min, Mean, and Max Point Query</h3>
+<p>
+  The endpoints here provide access to the total annual cumulative degree days
+  for several different degree day thresholds.
+</p>
+
+<h4>Endpoints</h4>
+<table>
+  <thead>
+    <tr>
+      <th>Endpoint</th>
+      <th>Threshold</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>heating</td>
+      <td>degree days (below 65&deg;F)</td>
+    </tr>
+    <tr>
+      <td>below_zero</td>
+      <td>degree days below 0&deg;F</td>
+    </tr>
+    <tr>
+      <td>thawing_index</td>
+      <td>degree days above freezing (above 32&deg;F)</td>
+    </tr>
+    <tr>
+      <td>below_zero</td>
```

Comment:
Should be freezing_index I think.

Reply:
Fixed in 52554fe79ce961400415f1113fab7293b18b90ab.